book/src/semantics/name-resolving.md

```diff
@@ -0,0 +1,76 @@
+# Name resolving
+
+Because PRQL primarily handles relational data, it has specialized scoping rules for referencing columns.
+
+## Scopes
+
+For pruposes of this book, a scope is the collection of all names one can reference from a specific point in the program.
+
+In PRQL, names in the scope are composed from namespace and variable name which are spearated by a dot, similar to SQL.
+Namespaces can contain many dots, but variable names cannot.
+
+> **Example**
+>
+> Name `my_table.some_column` is a variable `some_column` from namespace `my_table`.
+>
+> Name `foo.bar.baz` is a variable `baz` from namespace `foo.bar`.
```

Comment:
````suggestion
```admonish example
Name `my_table.some_column` is a variable `some_column` from namespace `my_table`.

Name `foo.bar.baz` is a variable `baz` from namespace `foo.bar`.
```
````